README.md

```diff
@@ -61,7 +61,6 @@ checkoutInstance.unmount()
 
 ```javascript
 function mySuccessHandler(successInfo) {
-  checkoutInstance.unmount()
   /** Show your own thank you message and/or do something with the payment info. */
 }
```

Comment:
I think there is some changes to be done in the README, like for example the handler onClose